### DISCUSSION TITLE
Some might benefit from it, but not me. In fact, I don't follow anybody...

### DISCUSSION THREAD

**Discussion by firminochangani** [community/community#13432] [Other Feature Feedback, Questions, & Ideas] [answered]
I don't follow anybody to avoid having my Github's home page polluted.

Kind regards,
Firmino

**Comment by jfullstackdev** [comment, marked as answer]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by firminochangani** [reply]
Much appreciated;
